doc/releases_and_migration/migration_guide_v280.rst

```diff
@@ -80,3 +80,53 @@ The new PAL serial bus implementation includes the following changes:
 
   .. note::
     It is recommended to disable the SPI bus instance in the devicetree when it is selected by the ``CONFIG_SIDEWALK_NRFX_SPI_INSTANCE_ID`` Kconfig option.
+
+
+.. _migration_guide_v280_new_libraries:
+
+Sidewalk v1.17 libraries
+************************
+
+The new Sidewalk libraries version 1.17 introduce metrics and capability features, along with fixes for known issues in previous versions.
+
+New end device characteristics
+==============================
+
+A new characteristic has been added to the Sidewalk end device configuration. This characteristic includes device type, power source, and qualification ID. To add this information, the developer must fill the dev_ch fields in the sid_config structure before the Sidewalk stack starts.
+
+An example of device characteristics is shown below:
+
+.. code-block:: C
+
+	struct sid_end_device_characteristics dev_ch = {
+		.type = SID_END_DEVICE_TYPE_STATIC,
+		.power_type = SID_END_DEVICE_POWERED_BY_BATTERY_AND_LINE_POWER,
+		.qualification_id = 0x0001,
+	};
+
+	sid_ctx.config = (struct sid_config){
+		.dev_ch = dev_ch,
+		.callbacks = &event_callbacks,
+		.link_config = app_get_ble_config(),
+		.sub_ghz_link_config = app_get_sub_ghz_config(),
+  };
+
+
+New Sidewalk ID option
+======================
+
+A new option to fetch the Sidewalk ID for the device has been introduced. Since the Sidewalk ID is assigned to the device post-registration, this option only works after the device has been registered.
+
+The Sidewalk ID can be retrieved using the CLI (DUT) option subcommand -gsi.
+
+.. code-block:: console
+
+  uart:~$ sid last_status 
+  [00:00:51.950,347] <inf> sid_cli: Device Is registered, Time Sync Fail, Link status: {BLE: Down, FSK: Down, LoRa: Down}
+  uart:~$ sid option -gsi
+  [00:00:55.582,641] <inf> sid_cli: sid_option returned 0 (SID_ERROR_NONE); SIDEWALK_ID: BFFFFFABCD
+
+Fixes uplink buffer
+===================
+
+The first uplink message after Link Up payload is now correct. The introduced workaround with pending_message_list can be removed.
```

Comment:
Is this something that should be in the migration guide? Sounds more like a resolved known issue but it seems it wasn't added in the ncs documentation (https://docs.nordicsemi.com/bundle/ncs-2.7.0/page/nrf/releases_and_maturity/known_issues.html)

Reply:
It should be mentioned in known issues, agrre.
I mention in here, because `pending_message_list` is part of sample (user aplication code), and can be safly removed during the migration

Reply:
So if we're going to include it in the ncs known issues would it make sense from your perspective to phrase it this way (?)
```suggestion
Fixing uplink buffer
===================

Upon migration, you can remove the ``pending_message_list`` workaround, as the first uplink message after Link Up payload has been fixed in the <Jira issue number>. 
For the issue details, see <link to known issues ncs doc>.
```

Reply:
Maybe replace `<Jira issue number>` with `v2.8.0`? jira issues are not visible for gihub users outside Nordic.
Besides that, it sounds good to me.

Reply:
Hmm, the reason I'd prefer to provide the issue number is not so much for customers to be able to see them in jira, but because all issues in "known issues" section have it as a reference number so it will be easier to find that issue on the list should anyone want to read about it